vision/ui/main_window.py

```diff
@@ -0,0 +1,67 @@
+from tkinter import*
+ 
+# creating button for autonomous mapping
+window = Tk()
+window.title("welcome")
+window.geometry('300x300')
+window.configure(width =50,height = 50, bg='blue')
+ 
+lbl = Label(window, text = "Autonomous Mapping",  bg = 'red', fg ='white')
```

Comment:
The code here is a bit repetitive, so it's a good practice to create a helper method like `add_button` that will handle adding the label and button for you. Then you could just keep calling that method to create all the buttons.